### DISCUSSION TITLE
Spree Adaptive Dark Mode

### DISCUSSION THREAD

**Discussion by MatthewKennedy** [spree/spree#10690] [Ideas]
Would anyone be interested in a pure CSS adaptive dark mode for the Spree admin UI?

[See Link](https://youtu.be/qcPTbqvx004)

**Comment by damianlegawiec** [comment]
I think that it would be an amazing addition :)

**Reply by MatthewKennedy** [reply]
I'll crack on with this today and see how it goes.
